### ISSUE TITLE
Remove `UIManager.setLayoutAnimationEnabledExperimental(true)` or make it optional

### ISSUE THREAD

**Issue by cip123** [fivecar/react-native-draglist#40]
Hi, and thanks for creating the library. It's very light and exactly what I needed.

However,  since I added the library to the project, I started getting Android crashes. 

I dug down in the code, and it seems like the following line causes them:

```
if (Platform.OS === "android") {
  if (UIManager.setLayoutAnimationEnabledExperimental) {
    UIManager.setLayoutAnimationEnabledExperimental(true);
  }
}
```

**Comment by fivecar** [comment]
I've not had a chance to look at this yet -- but are you saying this never works in Android? If you run the example app (eg. with an Android emulator), does it crash for you? The example app seems to run fine for me...

**Comment by cip123** [comment]
No, the tricky part is that it only crashes on some devices. It never fails on the simulator, but it crashes on real devices. I could reproduce it on a Samsung Galaxy phone, and I got numerous crash reports from Google Play store, which went away after I removed that line.

**Comment by fivecar** [comment]
Ok - those lines are directly what's in the [official React Native docs](https://reactnative.dev/docs/animations.html#layoutanimation-api).

But I don't think we use `LayoutAnimation` anymore, so I'll remove the lines.

**Comment by fivecar** [comment]
Released now in [3.6.2](https://github.com/fivecar/react-native-draglist/compare/3.6.1...3.6.2). Thanks again, @cip123!
